config-examples/simple-config-example.json

```diff
@@ -0,0 +1,16 @@
+{
+  "max-concurrent-indexers" : 2,
+  "dbpath" : "data",
+  "title" : "Hound",
+  "health-check-uri" : "/healthz",
+  "vcs-config" : {
+    "git": {
+      "detect-ref" : true
+    }
+  },
+  "repos" : {
+    "LocalFolder" : {
+      "url" : "file:////home/rfan/tensorflow"
```

Comment:
I mentioned this above, but I think `default-config.json` does the job of providing a simple config example. It also uses Hound's own source code on Github, so we know it'll work out of the box for most people. I believe it's used in our Dockerfile, so it might make sense to leave it (as well as the example config) in the root of the repository for now.

Reply:
I will remove this configuraton file and use the original example config.